run.go

```diff
@@ -135,18 +135,19 @@ func HandleError(err error) string {
 	return errStr
 }
 
-func RunCmd(ctx context.Context, root *cobra.Command) error {
+func RunCmd(ctx context.Context, rootCmd *cobra.Command) error {
 	p := make(map[string]string)
 	ctx = internal.ContextWithPersistedNames(ctx, p)
 	ctx, cancel := context.WithCancel(ctx)
 	handleInterrupt(ctx, cancel)
-	return root.ExecuteContext(ctx)
+	return rootCmd.ExecuteContext(ctx)
 }
 
 func handleInterrupt(ctx context.Context, cancel context.CancelFunc) {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt, os.Kill)
 	go func() {
+		defer close(c)
```

Comment:
Is it necessary to close the channel?

Reply:
Since channel `c` is not being used by any other part of the application, we can remove closing channel action. [7a991c2](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/7a991c20b78270f67c8e763f4727f6ef1e3a3463).